package.json

```diff
@@ -84,11 +83,14 @@
     "jest": "^28.1.0",
     "prettier": "^2.5.1",
     "prompts": "^2.4.2",
+    "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "ts-jest": "^28.0.2",
     "ts-node": "^10.4.0",
-    "typescript": "^4.6.4",
-    "zod": "^3.14.2"
+    "typescript": "^4.6.4"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@api3/airnode-validator": "^0.7.2",
+    "zod": "^3.14.2"
```

Comment:
You're encouraged to be curious :muscle: 

I added Zod parsing to `readOperationsRepository` - and reading operations is a production function.

A while back we had a bug in Telemetry/Monitoring that was due to the data of operations being different to the schema of operations. We weren't validating the result of loading the schema (which is a recursive JSON.parse function).

We were comparing the deviation threshold against the actual deviation, but the deviation threshold had moved. The if statement was therefore comparing an `undefined` value against a number, which didn't throw any exceptions.

Reply:
I should also note that this doesn't have a negative impact on the Explorer project because they read operations and then post-process it into an intermediate format - so Zod doesn't make its way to the browser.